ts/WoltLabSuite/Core/Acp/Ui/User/Action/AbstractUserAction.ts

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
I'm wondering if this file should be named:

```
ts/WoltLabSuite/Core/Acp/Ui/User/Action.ts
```

instead.

Reply:
What about `ts/WoltLabSuite/Core/Acp/Ui/User/Action/AbstractAction.ts` instead?

I think it would be quite misleading if the Abstract Action is placed in a different folder than the Actions themselves.

Reply:
```
ts/WoltLabSuite/Core/Acp/Ui/User/Action/Abstract.ts
```

This is consistent with other implementations and avoids the ambiguous `Action`.